lib/ReactTags.js

```diff
@@ -56,6 +56,43 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.handlePaste) {
```

Comment:
I'd still advocate sticking with React conventions and calling this `onPaste`, but take that as just an opinion of a fellow react component dev rather than anything more than that.